.codecov.yml

```diff
@@ -21,3 +21,5 @@ coverage:
     patch:
       default:
         target: 0 # we don't require any specific coverage for each PR
+
+            
```

Comment:
Unrelated changes

Reply:
ya, those are unrelated, nothing changed there.

Reply:
Then they shouldn't be in the PR

Reply:
removed.